kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -28,26 +28,43 @@ def airflow_commands():
 @click.option("-e", "--env", default="local")
 @click.option(
     "-t",
-    "--target-dir",
+    "--target-path",
     "target_path",
-    type=click.Path(writable=True, resolve_path=True, file_okay=False),
-    default="./airflow_dags/",
+    default="./airflow_dags/{package_name}_dag.py",
+)
+@click.option(
+    "-j",
+    "--jinja-file",
+    "jinja_template",
+    type=click.Path(exists=True, readable=True, resolve_path=True, file_okay=True),
+    default=Path(__file__).parent / "airflow_dag_template.j2",
 )
 @click.pass_obj
 def create(
-    metadata: ProjectMetadata, pipeline_name, env, target_path
-):  # pylint: disable=too-many-locals
+    metadata: ProjectMetadata,
+    pipeline_name,
+    env,
+    target_path,
+    jinja_template,
+):  # pylint: disable=too-many-locals,too-many-arguments
     """Create an Airflow DAG for a project"""
-    loader = jinja2.FileSystemLoader(str(Path(__file__).parent))
-    jinja_env = jinja2.Environment(autoescape=True, loader=loader, lstrip_blocks=True)
-    jinja_env.filters["slugify"] = slugify
-    template = jinja_env.get_template("airflow_dag_template.j2")
+    jinja_template = Path(jinja_template)
```

Comment:
```suggestion
    jinja_file = Path(jinja_file).resolve()
```
See https://stackoverflow.com/questions/42513056/how-to-get-absolute-path-of-a-pathlib-path-object for why `resolve` is preferred to `absolute`.